### PR TITLE
NGSTACK-813 add configuration for specifing inline vs download for mime type

### PR DESCRIPTION
**Pull request by ljacmi**
https://netgen.atlassian.net/browse/NGSTACK-813

### CODE REVIEW

**Review by iherak — APPROVED** [review]

File: bundle/Controller/Download.php [hunk 1]
```diff
@@ -16,29 +17,37 @@
...
+    ) {
+        $this->inlineMimeTypes = $this->configResolver->getParameter('download.show_inline', 'ngsite');
+    }
 
     /**
      * Downloads the binary file specified by content ID and field ID.
      *
      * Assumes that the file is locally stored
      *
+     * If isInline is less than 0, behaviour specified by configuration is observed.
```
Comment (emodric) [inline bundle/Controller/Download.php:43]: This is not true any more. Should be `If $isInline is null`.

File: bundle/Controller/Download.php [hunk 2]
```diff
@@ -16,29 +17,37 @@
 use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
+use function in_array;
 use function str_replace;
 
 final class Download extends Controller
 {
+    private array $inlineMimeTypes;
```
Comment (emodric) [inline bundle/Controller/Download.php:25]: Needs a fix for PHPStan. (`string[]` PHPDoc)

File: bundle/Controller/Download.php [hunk 3]
```diff
@@ -81,7 +96,7 @@ public function __invoke(Request $request, int|string $contentId, int|string $fi
         $response = new BinaryStreamResponse($binaryFile, $ioService);
         $response->setContentDisposition(
             (bool) $isInline ? ResponseHeaderBag::DISPOSITION_INLINE :
-            ResponseHeaderBag::DISPOSITION_ATTACHMENT,
+                ResponseHeaderBag::DISPOSITION_ATTACHMENT,
```
Comment (emodric) [inline bundle/Controller/Download.php:99]: Unintended change?

File: bundle/Controller/Download.php (hunk 2 above)
Comment (ljacmi) [reply bundle/Controller/Download.php:28]: Done

**Comment by emodric** [comment]
Thanks @ljacmi !

**Pull request merged**